jobs/service-fabrik-broker/templates/bin/health_check.erb

```diff
@@ -1,13 +1,27 @@
 #!/bin/bash
 
-nc -z $(hostname -I) <%= p('external.port') %>
+# Checking if HA setup needs to be done or not.
+ha_enabled=<%= p('ha_enabled') %>
```

Comment:
I guess this is where you are using the flag. Instead of throwing an exception for a misconfigured setup, can we not implicitly say that it is an HA setup looking at the number of instances?

Reply:
https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/pull/73#discussion_r188589756